test/gpu/jit.cpp

```diff
@@ -158,8 +164,20 @@ migraphx::src_file make_src_file(const std::string& name, const std::string& con
 
 TEST_CASE(simple_compile_hip)
 {
-    auto binaries = migraphx::gpu::compile_hip_src(
-        {make_src_file("main.cpp", write_2s)}, "", migraphx::gpu::get_device_name());
+    std::vector<migraphx::src_file> srcs;
+    std::transform(migraphx_kernels().begin(),
+                   migraphx_kernels().end(),
+                   std::back_inserter(srcs),
+                   [](auto&& p) {
+                       auto&& name = p.first;
+                       auto&& c    = p.second;
+                       auto path   = migraphx::fs::path{"migraphx"} / "kernels" / name;
```

Comment:
The paths can be removed if you use `char` instead,

Reply:
Done